src/lib/Preview.js

```diff
@@ -857,6 +862,22 @@ class Preview extends EventEmitter {
         // (access stats will not be incremented), but content access is still logged server-side for audit purposes
         this.options.disableEventLog = !!options.disableEventLog;
 
+        // Sets how previews of watermarked files behave.
+        // 'all' - Forces watermarked previews of supported file types regardless of collaboration or permission level,
+        //         except for `Uploader`, which cannot preview.
+        // 'any' - The default watermarking behavior in the Box Web Application. If the file type supports
+        //         watermarking, all users except for those collaborated as an `Uploader` will see a watermarked
+        //         preview. If the file type cannot be watermarked, users will see a non-watermarked preview if they
+        //         are at least a `Viewer-Uploader` and no preview otherwise.
+        // 'none' - Forces non-watermarked previews. If the file type cannot be watermarked or the user is not at least
+        //          a `Viewer-Uploader`, no preview is shown.
+        this.options.previewWMPref = options.previewWMPref || 'any';
```

Comment:
I don't think the "Pref" at the end adds much

Reply:
Wanted to distinguish some way between previewWM and downloadWM since one takes a string and one takes a boolean, but I'm open to dropping `Pref`

Reply:
ah makes sense, make the download one a boolean like verb by adding should or force to the front?

Reply:
+1 to prefixing the property name to help us distinguish boolean type

Reply:
I considered `shouldDownloadWM` and `forceDownloadWM` but I'm going to keep as `downloadWM` since download is already a verb and I interpret the option as downloadWatermarked, which implies true/false to me, whereas I interpret `previewWMPref` as previewWatermark(ed/ing)Preference, which implies options other than just true/false.